lib/code_corps_web/controllers/skill_controller.ex

```diff
@@ -1,24 +1,37 @@
 defmodule CodeCorpsWeb.SkillController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2, title_filter: 2, limit_filter: 2]
+  alias CodeCorps.{Skill, User, Helpers.Query}
 
-  alias CodeCorps.Skill
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
+  plug CodeCorpsWeb.Plug.IdsToIntegers
 
-  plug :load_and_authorize_resource, model: Skill, only: [:create]
-  plug JaResource
-
-  @spec model :: module
-  def model, do: CodeCorps.Skill
+  @spec index(Conn.t, map) :: Conn.t
+  def index(%Conn{} = conn, %{} = params) do
+    with skills <- Skill
```

Comment:
I wasn't able to find a community standard for a `with` clause which uses piping. This felt right, but I'm open to other suggestions 😄

Reply:
Simplest trick I can think off would be to add a `defp load_skills(%{} = params)` into the controller and call 

```Elixir
with skills <- params |> load_skills() do
```